addons/portal/static/src/scss/portal.scss

```diff
@@ -182,6 +182,12 @@ hr {
     background-color: inherit;
 }
 
+// IE 11 hack
+// http://browserhacks.com/#hack-d19e53a0fdfba5ec0f283ae86175a3af
+*::-ms-backdrop, :root, .flex-lg-grow-0 {
```

Comment:
@Julien00859 , I think that this selector it's too much generic.
It will affects all the `.flex-lg-grow-0` elements, even outside portal (the file is included in the frontend bundle). I suggest to restrict  to the sidebar only.